container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -380,53 +384,41 @@ instance_groups:
           privileged: true
 - name: mysql
   scripts:
-  - scripts/create_mysql_data_tmp.sh
+  - scripts/create_mysql_data_tmp.sh # Deprecated. Should go away with cf-mysql-release.
   - scripts/chown_vcap_store.sh
-  - scripts/patches/fix_mysql_advertise_ip.sh
+  - scripts/pxc/create_directories.sh
+  - scripts/patches/fix_mysql_advertise_ip.sh # Deprecated. Should go away with cf-mysql-release.
   - scripts/forward_logfiles.sh
-  - scripts/patches/enable_mysql_galera_bootstrap.sh
   - scripts/patches/fix_monit_rsyslog.sh
+  post_config_scripts:
+  - scripts/bpm_kube_dns.rb
   jobs:
   - name: global-properties # needs to be first so images use it for processing monit templates.
     release: scf-helper
   - name: patch-properties
     release: scf-helper
-  - name: mysql
-    release: cf-mysql
+  - name: pxc-mysql
+    release: pxc
     provides:
       mysql: {}
     properties:
       bosh_containerization:
         ports:
-        - name: mysql
+        - name: pxc-mysql
           protocol: TCP
-          internal: 13306
+          internal: 3306
         - name: galera-tcp
           protocol: TCP
           internal: 4567
-        - name: galera-udp
-          protocol: UDP
-          internal: 4567
-        - name: galera-ist
-          protocol: TCP
-          internal: 4568
-        - name: healthcheck
+        - name: galera-agent
           protocol: TCP
           internal: 9200
-        - name: xtrabackup
-          protocol: TCP
-          internal: 4444
-  - name: proxy
-    release: cf-mysql
-    provides:
-      proxy: {}
-    properties:
-      bosh_containerization:
         run:
           scaling:
             min: 1
-            max: 3
-            ha: 2
+            max: 7
```

Comment:
From the pxc release [doc](https://github.com/cloudfoundry-incubator/pxc-release#database-nodes):
_The number of mysql nodes should always be odd, with a minimum count of three, to avoid split-brain._
So now since `ha` was deemed 3, `max` had to be a higher number so either 5 or 7 would work.

Reply:
Then we should also have [`must_be_odd`](https://github.com/cloudfoundry-incubator/fissile/blob/0d8469bb8518a5457a9a3bb7997ffd386739f17b/model/role_run.go#L54). Not sure why we can't support, say, 9 nodes, though? Or 32787? 65535? (That is: do we _need_ a max?)

Reply:
I can definitely add `must_be_odd`, if you can confirm I am using it correctly:
```
run:
    scaling:
       min: 1
       max: 7
       ha: 3
       # To avoid split-brain.
       must_be_odd: true
```
In theory, there's no real reason why we can't scale up the nodes beyond `max`, it should only be restricted by the amount of resources we have on the cluster. For now, `max` is just kept as-is(with an increased count value) but it can be removed.

Reply:
I believe that's how it's meant to be used, yes.

We can deal with the `max` separately; as I mentioned, it's not like `max: 3` was much better.

Reply:
`must_be_odd` added and set to `true`.

Reply:
How about removing max?

Reply:
as @mook-as mentioned let's handle it in a different task.